openapi/swagger.yml

```diff
@@ -109,6 +147,16 @@ definitions:
         type: number
         format: float
         description: Budget used by this read request
+  DatasetReleaseRequest:
```

Comment:
This assumes the original dataset was private? Should it?

Reply:
resolved